CHANGELOG.md

```diff
@@ -41,6 +41,7 @@
 - Remove duplicate HTTP Client Errors on iOS ([#4250](https://github.com/getsentry/sentry-react-native/pull/4250))
 - Replay `maskAll*` set to `false` on iOS kept all masked ([#4257](https://github.com/getsentry/sentry-react-native/pull/4257))
 - Add missing `getRootSpan`, `withActiveSpan` and `suppressTracing` exports from `@sentry/core`, and `SeverityLevel` export from `@sentry/types` ([#4254](https://github.com/getsentry/sentry-react-native/pull/4254), [#4260](https://github.com/getsentry/sentry-react-native/pull/4260))
+- `browserReplayIntegration` should not be included by default ([#4270](https://github.com/getsentry/sentry-react-native/pull/4270))
```

Comment:
It's good that we are in beta Replay is on Beta :D 

We should add a small snippet on the changelog to show how to load mobileReplayIntegration on React Native Web since the behavior has changed.